test/e2e/create_test.go

```diff
@@ -241,4 +241,10 @@ var _ = Describe("Podman create", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To((Equal(0)))
 	})
+
+	FIt("podman create --authfile with nonexist authfile", func() {
+		session := podmanTest.PodmanNoCache([]string{"create", "--authfile", "/tmp/nonexist", "--name=foo", "nginx"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).To(Not(Equal(0)))
+	})
```

Comment:
as painful as it will be, it would be good to add a similar test to each of the other commands.